src/__tests__/app/email-template-submitted/__snapshots__/page.test.tsx.snap

```diff
@@ -62,13 +62,14 @@ exports[`EmailTemplateSubmitted 1`] = `
         If you've already onboarded
       </h3>
       <p>
+        Once you've submitted all your templates 
         <a
           href="https://nhsdigitallive.service-now.com/nhs_digital?id=sc_cat_item&sys_id=6208dbce1be759102eee65b9bd4bcbf5"
           id="servicenow-link"
+          target="_blank"
```

Comment:
I don't know if this might be relevant here -  https://owasp.org/www-community/attacks/Reverse_Tabnabbing

Reply:
updated with `rel=”noopener noreferrer”` as advised to prevent this.

Reply:
If you are adding a simple attribute then just add it to all places where we open in a new tab, should be quick.

Reply:
Added to all place where we open a new tab 👍